test/readme.md

```diff
@@ -22,3 +22,8 @@ If a test suite already exists that relates closely to the item being tested (e.
 
 - When checking for a condition that might take time, ensure it is waited for either using the browser `waitForElement` or using the `check` util in `next-test-utils`.
 - When applying a fix, ensure the test fails without the fix. This makes sure the test will properly catch regressions.
+
+## Helpful env variables
+
+- When investigating failures in isolated tests you can use `NEXT_TEST_SKIP_CLEANUP=1` to prevent deleting the temp folder created for the test.
```

Comment:
Would be helpful provide some clarity on results (e.g "This reduces test time / ignores errors when writing tests")

Reply:
Good point added a bit more clarity/info in https://github.com/vercel/next.js/pull/38566/commits/4455e106bf86a36fdcd2296358e635284553aef4